templates/reputation/contracts/ReputationTemplate.sol

```diff
@@ -0,0 +1,97 @@
+pragma solidity 0.4.24;
+
+import "@aragon/templates-shared/contracts/BaseTemplate.sol";
+
+
+contract ReputationTemplate is BaseTemplate {
+    string private constant ERROR_MISSING_TOKEN_CACHE = "REPUTATION_MISSING_TOKEN_CACHE";
+    string private constant ERROR_INVALID_HOLDERS_STAKES_LEN = "REPUTATION_INVALID_HOLDERS_STAKES_LEN";
```

Comment:
Super nitpick, but I would use `REP_` prefix to make these ones shorter (sorry, I have had some much trouble with EIP-170 recently that I guess I'm getting obsessed, haha)